sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -92,13 +92,12 @@ async def receive_frame(self, *args, **kwargs):
                 decoded = decode_empty_frame(header)
             else:
                 decoded = decode_frame(payload)
-            # TODO: Catch decode error and return amqp:decode-error
             #_LOGGER.info("ICH%d <- %r", channel, decoded)
```

Comment:
not part of the TODO, but do we want to uncomment or delete this line?

Reply:
Ill uncomment it for now and we can review during the 'logging review' portion of the work